src/coreComponents/mesh/generators/InternalWellGenerator.hpp

```diff
@@ -59,7 +59,7 @@ class InternalWellGenerator : public MeshGeneratorBase
   /**
    * @brief Default destructor.
    */
-  virtual ~InternalWellGenerator() override;
+  virtual ~InternalWellGenerator();
```

Comment:
I think that
```suggestion
  ~InternalWellGenerator() override;
```
would be the preferred combination.